src/ngx_http_lua_shdict.c

```diff
@@ -2983,6 +2983,24 @@ ngx_http_lua_ffi_shdict_set_expire(ngx_shm_zone_t *zone, u_char *key,
 
     return NGX_OK;
 }
+
+
+#if nginx_version >= 1011007
```

Comment:
Style: better add 4 spaces between `#` and `if` to indicate the nested level. Let's also do this for the corresponding `#endif` below?

Reply:
done